common/repo/Repo.java

```diff
@@ -81,5 +81,10 @@
   String getRemoteURL();
   /** Checks if there are commits added since master, or any uncommitted files */
   boolean hasChanges(String branch);
+  /**
+   * Gets patch(diff) between current file and the file of the base branch(usually "master"). It's
```

Comment:
See previous comment - isn't this between the current file and the file in the passed `commitOrBranch`?

Reply:
Done.